{{cookiecutter.project_name}}/docs/conf.py

```diff
@@ -84,10 +102,17 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "furo"
+html_theme = "sphinx_book_theme"
 html_static_path = ["_static"]
+html_title = project_name
+
+github_repo = "https://github.com/" + html_context["github_user"] + "/" + project_name
```

Comment:
```suggestion
```

Get it from project metadata, see previous suggestion.

Reply:
Done.